genie-web/src/main/java/com/netflix/genie/grpc/services/impl/GreeterServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+ package com.netflix.genie.grpc.services.impl;
+
+import com.netflix.genie.grpc.interceptors.SimpleLoggingInterceptor;
+import com.netflix.genie.protogen.GreeterServiceGrpc;
+import com.netflix.genie.protogen.HelloReply;
+import com.netflix.genie.protogen.HelloRequest;
+import io.grpc.stub.StreamObserver;
+import net.devh.springboot.autoconfigure.grpc.server.GrpcService;
+
+/**
+ * Example implementation of the Greeter service definition.
+ */
+@GrpcService(
+    value = GreeterServiceGrpc.class,
+    interceptors = {
+        SimpleLoggingInterceptor.class,
+    }
+)
+public class GreeterServiceImpl extends GreeterServiceGrpc.GreeterServiceImplBase {
```

Comment:
Is there any reason to actually commit these things into the repository? We'll never actually use this it's like checking in hello world right?

Reply:
Right now, the only value is testing that the build works (e.g., generates protobufs, Spring configures the service, etc).
I agree we should remove them as soon as we have some real services or messages.